src/IECore/CurveAlgo.cpp

```diff
@@ -349,6 +349,21 @@ struct  CurvesVaryingToVertex
 	const CurvesPrimitive *m_curves;
 };
 
+struct  AverageValueFromVector
```

Comment:
This seems to belong in the previous commit rather than the "bindings & tests" commit.